backend/graphql/service/product.service.ts

```diff
@@ -0,0 +1,14 @@
+import { ProductModel } from '../schema/product.schema'
+import { type inputProduct } from '../types/product.type'
+
+class ProductService {
+  findProducts() {
+    return ProductModel.find()
+  }
+
+  async findSingleProduct(input: inputProduct) {
+    return await ProductModel.findOne(input)
+  }
+}
+
+export default ProductService
```

Comment:
Let’s always use exports, default exports can be named anything when imported which can make maintenance harder